care/facility/migrations/0001_initial_squashed.py

```diff
@@ -1524,7 +1524,7 @@ class Migration(migrations.Migration):
                     ),
                 ),
                 (
-                    "ventilator_fi02",
+                    "ventilator_fio2",
```

Comment:
You should instead be creating a new migration that renames the field by using `python manage.py makemigrations`.

cc: @sainak